artemis-server/src/main/java/org/apache/activemq/artemis/core/persistence/impl/journal/LargeServerMessageImpl.java

```diff
@@ -363,13 +363,17 @@ public String toString() {
    }
 
    @Override
-   public void validateFile() throws ActiveMQException {
+   public void validateFile() {
       this.ensureFileExists(true);
    }
 
-   public void ensureFileExists(boolean toOpen) throws ActiveMQException {
+   public void ensureFileExists(boolean toOpen) {
       synchronized (largeBody) {
-         largeBody.ensureFileExists(toOpen);
+         try {
+            largeBody.ensureFileExists(toOpen);
+         } catch (Exception e) {
+            logger.debug(e.getMessage(), e);
+         }
```

Comment:
Seems odd to swallow this...so ensureFileExists() no longer ensures the file exists? Meaning validateFile() isnt validating much?